chpl/chpl-api/e2e/collections/developer-controller.postman_collection.json

```diff
@@ -672,7 +672,8 @@
+                            ncy Services Solution\",\"reportFileLocation\":null,\"contact\":null,\"owner\":{\"developerId\":1926,\"developerCode\":\"2925\",\"name\":\"DSS, Inc.\",\"website\":null,\"selfDeveloper\":false,\"address\":null,\"contact\":null,\"lastModifiedDate\":null,\"deleted\":null,\"transparencyAttestations\":[],\"statusEvents\":[],\"status\":null},\"ownerHistory\":[],\"lastModifiedDate\":\"1592858752672\",\"versions\":[{\"versionId\":8134,\"version\":\"v. 3.1\",\"details\":null,\"lastModifiedDate\":\"1592858752801\",\"listings\":[{\"id\":10411,\"chplProductNumber\":\"15.04.04.2925.JESS.03.00.0.200619\",\"lastModifiedDate\":1600104562632,\"edition\":\"2015\",\"certificationDate\":1592524800000,\"certificationStatus\":\"Active\",\"curesUpdate\":false,\"acb\":{\"id\":3,\"acbCode\":\"04\",\"name\":\"Drummond Group\",\"website\":\"http://www.drummondgroup.com\",\"address\":{\"addressId\":2,\"line1\":\"13359 North Hwy. 183\",\"line2\":\"Suite B 406-238\",\"city\":\"Austin\",\"state\":\"Texas\",\"zipcode\":\"78750\",\"country\":\"USA\"},\"retired\":false,\"retirementDate\":null}}]}]}]}"
```

Comment:
This shouldn't be in new line. looks accidently

Reply:
not sure why it shows up the way it does here, it doesn't show up with line break for me locally

Reply:
but you are right, that was unintentional. I am going to figure out how to resolve that but have to away atm

Reply:
pushed another commit to fix it